tests/test_plot.py

```diff
@@ -620,6 +620,12 @@ def pdf(x, a=1 / np.sqrt(2 * np.pi), x0=0, sigma=1, offset=0):
 
     fig = plt.figure()
 
-    assert h.plot_pull(pdf, fit_fmt=r"{name} = {value:.3g} $\pm$ {error:.3g}")
+    assert h.plot_pull(
+        pdf,
+        eb_color="black",
+        fp_color="blue",
+        ub_color="lightblue",
```

Comment:
The default  `matplotlib` proc cycle colors are quite ugly for a plot like this, so choosing these for now so that the different components are actually visible.